app/jobs/submit_patron_request_job.rb

```diff
@@ -80,7 +80,8 @@ def place_title_hold(patron_request)
     hold_request_data = FolioClient::HoldRequestData.new(pickup_location_id: patron_request.pickup_service_point.id,
                                                          patron_comments: patron_request.request_comments,
                                                          expiration_date: patron_request.needed_date.to_time.utc.iso8601)
-    folio_response = folio_client.create_instance_hold(patron_request.patron.id, patron_request.instance_hrid, hold_request_data)
+    # Use instance_id instead of instance_hrid b/c the former is what is required, the latter will leave off the "a" at the beginning
```

Comment:
@hudajkhan  I think we can get rid of this. create_instance_hold doesn't allow for hrid so that might be confusing without the commit history. https://github.com/sul-dlss/sul-requests/blob/562a92559c342d5466ce8801b73f35be21b89a8f/app/services/folio_client.rb#L224